2022-02_sprint/schedule.md

```diff
@@ -17,7 +17,7 @@
 
 | Name of Event                 | Date of Event        | Time of Event | Duration      | Registration form  |
 |-------------------------------|----------------------|---------------|---------------|---------------------|
-| {ref}`Pre Sprint <sprint/schedule>`| February 11, 2022 |  3pm UTC    | 1 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283765923/) |
-| {ref}`sprint` slot 1          | February 18, 2022    |  7pm UTC      | 2 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283178769/) |
-| {ref}`sprint` slot 2          | February 19, 2022    |  10am UTC     | 2 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283178769/) |
-| {ref}`Post Sprint <sprint/schedule>`| February 25, 2022 |  3pm UTC   | 1 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283766699/) |
+| Pre Sprint| February 11, 2022 |  3pm UTC    | 1 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283765923/) |
+| {ref}`Sprint <sprint>` slot 1          | February 18, 2022    |  7pm UTC      | 2 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283178769/) |
+| {ref}`Sprint <sprint>` slot 2          | February 19, 2022    |  10am UTC     | 2 hours       | [meetup event](https://www.meetup.com/data-umbrella/events/283178769/) |
```

Comment:
Let's remove the cross-references altogether. This is the old schedule and I think that having links to the constant section of the website as if related to specific events listed there is confusing.

This file also needs the headings updated. I think they should be: `# Schedule` on top, `## Webinars` right below, above the webinar table, then replace the `# Sprint` one by `## Sprint events`

Reply:
Done!